src/app/components/media-viewer-wrapper/media-viewer-wrapper.component.scss

```diff
@@ -0,0 +1,220 @@
+@import "../../../../node_modules/@hmcts/media-viewer/assets/aui-styles.scss";
+
+html {
+    overflow: hidden;
+}
+
+.pdfViewer .page {
+    direction: ltr;
+    margin: 0 auto;
+    position: relative;
+    overflow: visible;
+    background-clip: content-box;
+    border-image: url('/assets/images/shadow.png') 9 9 repeat;
+    background-color: white;
+  }
+
+  .pdfViewer .canvasWrapper {
+    box-shadow: 0 0 3px #bbb;
+  }
+  .pdfViewer .page {
+    margin-bottom: 10px;
+  }
+
+  .annotationLayer {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+  }
+
+  .pdfViewer .page {
+    border-image: none;
+  }
+
+  .textLayer {
+    position: absolute;
+    left: 0;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    overflow: hidden;
+    opacity: 0.2;
+    line-height: 1.0;
+  }
+
+  .textLayer > div {
+    color: transparent;
+    position: absolute;
+    white-space: pre;
+    cursor: text;
+    -webkit-transform-origin: 0% 0%;
+    -moz-transform-origin: 0% 0%;
+    -o-transform-origin: 0% 0%;
+    -ms-transform-origin: 0% 0%;
```

Comment:
Shouldn't need to use vendor prefixes in scss with angular, as these will get added during build based on the config in the browserslist file